frontend/device/tolino/device.lua

```diff
@@ -0,0 +1,33 @@
+local logger = require("logger")
+local A, android = pcall(require, "android")  -- luacheck: ignore
+local AndroidDevice = require("device/android/device")
+
+local function yes() return true end
+local function no() return false end
+
+local TolinoDevice = AndroidDevice:new{
+    model = "Tolino"
+}
+
+function TolinoDevice:init()
+    AndroidDevice.init(self)
+    self.input.event_map = require("device/tolino/event_map")
+end
+
+-- Tolino Vision 5, hardware model "E70K00"
+local TolinoE70K00 = TolinoDevice:new{
+    model = "Tolino_E70K00",
+    hasEinkScreen = yes,
+    frontlight_settings = {
```

Comment:
even if that works, the proper abstraction is in android-luajit-launcher, for both epd and lights driver. Please do that there so you won't break other widgets (ie: the native frontlight widget doesn't understand your code here)

Reply:
> even if that works, the proper abstraction is in android-luajit-launcher

i basically copied it from `device/kobo/device` for future reference